contracts/RLPTest.sol

```diff
@@ -0,0 +1,107 @@
+pragma solidity ^0.4.23;
+import "./RLP.sol";
+
+
+/**
+ *	@title RLPMock contract is for RLP library contract
+ *
+ *	@notice It is used to test methods in RLP library contract
+ */
+contract RLPMock  {
+
+    constructor() public{}
+
+    /**
+     *  @notice Creates an RLPItem from an array of RLP encoded bytes
+     *
+     *  @param rlpEncodedData The RLP encoded bytes
+     *
+     *  @return length of the rlp data
+     *  @return memory pointer at which rlp data is present
+     */
+    function toRLPItem(bytes rlpEncodedData)
+        public
+        pure
+        returns
+        (uint /*length*/,
+        uint /*memory_pointer*/)
+    {
+        RLP.RLPItem memory item = RLP.toRLPItem(rlpEncodedData);
+        return (item._unsafe_length,item._unsafe_memPtr);
+    }
+
+    /**
+     *   @notice Creates an RLPItem from an array of RLP encoded bytes having
+     *
+     *   @param rlpEncodedData The RLP encoded bytes
+     *   @param strict Will throw if the data is not RLP encoded
+     *
+     *   @return length of the rlp data
+     *   @return memory pointer at which rlp data is present
+     */
+    function toRLPItemStrict(
+        bytes rlpEncodedData,
+        bool strict)
+        public
+        pure
+        returns
+        (uint /*length*/,
+        uint /*memory_pointer*/)
+    {
+        RLP.RLPItem memory item = RLP.toRLPItem(rlpEncodedData,strict);
+        return(item._unsafe_length,item._unsafe_memPtr);
+    }
+
+
+    /**
+     * @notice Get the list of sub-items from an RLP encoded list
+     * Warning: This is inefficient, as it requires that the list is read twice
+     *
+     * @param rlpEncodedData The RLP encoded bytes
+     *
+     * @return length of list
+     */
+    function toList(
+        bytes rlpEncodedData)
+        pure
+        public
+        returns(uint)
+    {
+        RLP.RLPItem memory item = RLP.toRLPItem(rlpEncodedData);
+        RLP.RLPItem[] memory list =  RLP.toList(item);
+        return list.length;
```

Comment:
I think this should return the list of sub-items. Returning the length of the list of sub-items does not provide me confidence that `toList` correctly returns an array of the components of `item`.